include/fi_util.h

```diff
@@ -410,4 +411,8 @@ int ofix_getinfo(uint32_t version, const char *node, const char *service,
 			int get_base_info, struct fi_info **info);
 char *ofi_strdup_less_prefix(char *name, char *prefix);
 char *ofi_strdup_add_prefix(char *name, char *prefix);
+
+int ofi_shm_map(struct util_shm* shm, const char* name, size_t size, int readonly, void** mapped);
```

Comment:
nit - please break line at close to 80 characters, and move the * next to the variable name rather than the structure name, to make it clearer to a reader that these are pointers